website/docs/r/schema_template.html.markdown

```diff
@@ -14,21 +14,24 @@ Manages MSO Schema Template
 
 ```hcl
 
-resource "mso_schema_template" "st1" {
-  schema_id    = mso_schema.schema1.id
-  name         = "Temp1"
-  display_name = "Temp1"
-  tenant_id    = mso_tenant.tenant1.id
+resource "mso_schema_template" "demo_template" {
+  schema_id     = mso_schema.demo_schema.id
+  name          = "Template1"
+  display_name  = "Template1"
+  tenant_id     = mso_tenant.demo_tenant.id
+  template_type = "aci_multi_cloud"
 }
 
 ```
 
 ## Argument Reference ##
 
-* `schema_id` - (Required) name of the schema.
-* `tenant_id` - (Required) Tenant-id to associate.
-* `name` - (Required) Name of the template.
-* `display_name` - (Required) Display name of the Template to be deployed on the site.
+* `name` - (Required) The name of the template.
+* `schema_id` - (Required) The schema-id where template is associated.
+* `tenant_id` - (Required) The tenant-id to associate with the template.
+* `display_name` - (Required) The display name of the template.
+* `template_type` - (Required) The template type of the template.
```

Comment:
Missing choices and default value when not provided.

Reply:
ok